packages/fluent/scss/wizard/_layout.scss

```diff
@@ -45,8 +45,6 @@
         }
 
         .k-wizard-content {
-            padding-inline: var( --kendo-wizard-content-padding-x, #{$kendo-wizard-content-padding-x} );
-            padding-block: var( --kendo-wizard-content-padding-y, #{$kendo-wizard-content-padding-x} );
```

Comment:
either set the variables to 0px and allow the option for customization or remove both the _layout styles and the variables as well.